src/webviews/mongoClusters/collectionView/components/toolbar/ToolbarViewNavigation.tsx

```diff
@@ -48,6 +110,23 @@ export const ToolbarViewNavigation = (): JSX.Element => {
                 pageNumber: 1,
             },
         });
+
+        trpcClient.common.reportEvent
+            .mutate({
+                eventName: 'pagination',
+                properties: {
+                    source: 'page-size',
+                    ui: 'button',
+                    view: currentContext.currentView,
+                },
+                measurements: {
+                    page: currentContext.currrentQueryDefinition.pageNumber,
+                    pageSize: pageSize,
+                },
+            })
+            .catch((error) => {
+                console.error('Failed to report pagination event:', error);
```

Comment:
If this is error then either it should show message to user or send to telemetry. If this is debug info, please use console.debug or log.